### ISSUE TITLE
Volunteer collaborators are needed

### ISSUE THREAD

**Issue by platosha** [platosha/angular-polymer#123]
We are looking for collaborators to work on this project. If you want to become a collaborator, please leave a comment below.

Collaborator responsibilities include:
- Responding to issues
- Reviewing and merging pull requests
- Development contributions: add features, fix bugs

**Comment by BorntraegerMarc** [comment]
I think we are already in contact :)

**Comment by MarkPieszak** [comment]
@platosha Is there a Slack channel or anything we can all talk in? 
Maybe we can make a channel in the Polymer Slack?

But yes, definitely would like to help out! 
Do you have a design doc or anything about how it's all put together currently, wanted to see if any private pieces of Core we're touching will be affected in 4.x in March. @platosha

**Comment by BorntraegerMarc** [comment]
Good idea. I was in the process of creating an own https://angular-polymer.slack.com. What makes more sense? To integrate in the existing polymer slack or a new one? In my opinion to get a channel in the existing polymer slack would be a bit better...

Friendly ping to @platosha :)

**Comment by MarkPieszak** [comment]
I think we should stay close to Polymer, we might as well be in there.

**Comment by BorntraegerMarc** [comment]
Agreed. I've already made a request in their slack for an own channel. I hope it's possible...

**Comment by platosha** [comment]
Works for me.

If using Polymer Slack is not possible, I would prefer to create a Gitter chat over Slack then. I like the gitter UI more, and also the webcomponents.org community chat is there.

**Comment by platosha** [comment]
@MarkPieszak We don’t have a design doc yet.

In short, pretty much all the current implementation is in the single file `src/polymer-element.ts`, and most of the job is done by the exported `PolymerElement(name: string)` function. The idea there is simple: it takes an element name from the argument, extracts the element’s prototype, generates a few directives based on the prototype, and returns them.

The only private part was `BrowserDomAdapter`, as far as I remember. As you probably know, the related part has already stopped working, and the renderer replacement is coming up.

**Comment by MarkPieszak** [comment]
Perfect yeah let me take a look at the implementation to see where I can help out!

**Comment by manekinekko** [comment]
Hi. I would like to help out. Btw, have you seen this interop work https://github.com/webcomponents/angular-interop?

**Comment by MarkPieszak** [comment]
Wassim! Now it's a party :beers:

**Comment by BorntraegerMarc** [comment]
@manekinekko Never seen it but seems cool. Did you try it for production use? And how does it differ to this implementation?

**Comment by manekinekko** [comment]
@MarkPieszak haha! good to see you here. I'm glad Michael Prentice sent me here ^^

@BorntraegerMarc the interop implementation is straightforward. My goal was to use vanilla  Polymer with raw Angular (>=2) without any glue code (or at as little as possible). So this interop repo tries to serve as an implementation reference. It's not a library.

FYI, I've also implemented another interop repo for Web Components (v0 spec) here: https://github.com/manekinekko/angular-wc-interop 

I'm was wondering what made you create this library? 

PS: I'm  OK to continue this discussion on slack/gitter.

**Comment by MarkPieszak** [comment]
2 way binding with polymer mainly. @platosha could answer that best

**Comment by Splaktar** [comment]
It also does some work to hook up Form events and inputs with Polymer web components. In the past, it has had issues with accessing the window object to get window.Polymer which doesn't play well with Universal. So in addition to switching to the new Renderer interface, it would be nice to get it working with Universal.

**Comment by MarkPieszak** [comment]
@Splaktar Yeah definitely, we want it to play well with Universal/web-workers for sure. I talked with @platosha the other day so I started looking into a few things to get everything going to get the project up to date with everything.

We'll be switching everything to use the `Renderer`
At the very least just ignore `window.Polymer` and bail out if it doesn't exist, so we won't be able to *render* anything on the server, but at least it'll appear on the client later.
The other main issue is AoT since you can't have functions like PolymerElement()

Maybe we can all meet up sometime next week?
Anton and I have been talking in the Polymer slack if you guys are in there, think he's trying to get a channel for this as well.

**Comment by Splaktar** [comment]
I'm in the Polymer Slack, but I can't keep up with things in general. A specific channel would be really helpful.

**Comment by BorntraegerMarc** [comment]
We just got the https://polymer.slack.com/messages/polymer-angular/ channel :)

**Comment by malkomich** [comment]
I would like to contribute to this project.

Can anyone tell me how to begin? Any straightforward issue to start working on it?

**Comment by rjcorwin** [comment]
@manekinekko Unfortunately that example is out of date.

- https://github.com/webcomponents/angular-interop/issues/3
- https://github.com/webcomponents/angular-interop/issues/4

I think they need to upgrade to Polymer 2 and base the example on Angular CLI.

**Comment by ZiruZanrgeiff** [comment]
Hello guys i would like to contribute. Nowadays I am using this component and I can't update my anguar cli for this reason. Well what do we need to do for make this possible? How can I help?
